### PR TITLE
Chronos cleanup

### PR DESCRIPTION
**Pull request by digitalyuki**
Greetings and thanks for the PR.  We have a few rules so that everyone enjoys your Pull Request.

## Changelog
Add new chronos-cleanup.service and chronos-cleanup.timer
## Notes

You must explain how any `docker run` executions your PR introduces handles log accumulation. 

Did you introduce any command that executes `docker run`?
*No*
Are there any dependencies on github.com/adobe-platform/infrastructure?
*No*
If so:
 
Is an update to the base stack required(rare)?
*No*
Is an A/B required?
*Yes*
Are you dependent on new secrets, or infrastructure in any way?
No, but if this PR is merged, we'll need to add in chronos authentication: https://github.com/adobe-platform/ethos-systemd/pull/46

### CODE REVIEW

**Comment by matthewdfuller** [comment]
@digitalyuki looks good. Two questions:

1. Did you make the sh script executable (chmod +x)? I can't tell from git, but that's bitten me in the past.
2. We are [going to be adding](https://github.com/adobe-platform/ethos-systemd/pull/46/files) user/pass auth to Chronos shortly. Is that going to prevent the API calls made here from working?

File: v1/util/chronos-cleanup.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+CHRONOS_URL="http://localhost"
+CHRONOS_PORT="4400"
+ALL_JOBS="$(curl -L -X GET ${CHRONOS_URL}:${CHRONOS_PORT}/scheduler/jobs | jq -r '.[] | [ .schedule, .name ]| join(",")' )"
+OLD_JOBS="$(echo "$ALL_JOBS" | grep '^R0/')"
+OLD_JOB_NAMES="$( echo "$OLD_JOBS" | awk -F ',' '{print $2}' )"
+if [[ "$OLD_JOB_NAMES" != "" ]] ; then
+  echo -e "Old jobs found:\n$OLD_JOB_NAMES\n"
```
Comment (mannytoledo) [inline v1/util/chronos-cleanup.sh]: This output will end up in sumologic at some point. Can we add `CHANNEL=chronos_cleanup` to the output. It at least gives us something obvious to filter on.

File: v1/util/chronos-cleanup.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  for job_name in $OLD_JOB_NAMES ; do 
+    echo "Deleting $job_name..."
```
Comment (mannytoledo) [inline v1/util/chronos-cleanup.sh]: Same here. `APP_NAME` might also be appropriate for either of these.

File: v1/util/chronos-cleanup.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    curl --write-out '  curl return code: %{http_code}\n' -L -X DELETE ${CHRONOS_URL}:${CHRONOS_PORT}/scheduler/job/${job_name}
+  done
+else 
+  echo "No old jobs to be deleted!"
```
Comment (mannytoledo) [inline v1/util/chronos-cleanup.sh]: Here too.

**Comment by digitalyuki** [comment]
@matthewdfuller : 
1) ethos-systemd/v1/util/chronos-cleanup.sh loads up as 755, similar to the rest of the scripts in the util/ directory. 
2) we'll need to add in the chronos authentication, currently we have chronos credentials available in secrets but chronos authentication is not yet enabled... i just verified that if i pass along credentials even if the url doesn't need it, it'll still pass through, so i can add that in shortly.

**Comment by matthewdfuller** [comment]
Perfect, thanks! 👍

**Pull request merged**